### PR TITLE
Remove IndexUtil#oneToZeroIndex(int)

### PR DESCRIPTION
**Pull request by Zhiyuan-Amos**

### CODE REVIEW

**Comment by CanIHasReview** [comment]
# v1

@Zhiyuan-Amos submitted v1 for review.

* [v1 1/1] [Remove IndexUtil\#oneToZeroIndex\(int\)](https://github.com/se-edu/addressbook-level4/pull/463/commits/ec5c329fd6b23cb2c9617032aac6fcfd85dff2ea)

([:books: Archive](https://github.com/se-edu/addressbook-level4/compare/5ccd6d185c69c6af44f01b45eab992d90a4358f3...ec5c329fd6b23cb2c9617032aac6fcfd85dff2ea))
<details>
<summary>Checkout this PR version locally</summary>

```sh
git fetch https://github.com/se-edu/addressbook-level4.git refs/pr/463/1/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you wish to fetch this PR to.

</details>

**Comment by CanIHasReview** [comment]
# v2

@Zhiyuan-Amos submitted v2 for review.

* [v2 1/1] [Remove IndexUtil\#oneToZeroIndex\(int\)](https://github.com/se-edu/addressbook-level4/pull/463/commits/ccc33948a9c40af85554a7676cfecdec026f1c78)

([:books: Archive](https://github.com/se-edu/addressbook-level4/compare/dfe3b25eb274119f2d0605fa266877597a3341c6...ccc33948a9c40af85554a7676cfecdec026f1c78)) ([:chart_with_upwards_trend: Interdiff between v1 and v2](https://github.com/se-edu/addressbook-level4/compare/15d1f729f9d51383be93850143a1fd35fb2276b5~1...15d1f729f9d51383be93850143a1fd35fb2276b5))
<details>
<summary>Checkout this PR version locally</summary>

```sh
git fetch https://github.com/se-edu/addressbook-level4.git refs/pr/463/2/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you wish to fetch this PR to.

</details>

File: src/test/java/guitests/EditCommandTest.java [hunk 1]
```diff
@@ -58,26 +61,25 @@ public void edit_notAllFieldsSpecified_success() throws Exception {
...
+        Person personToEdit = expectedPersonsList[addressBookIndex.getZeroBased()];
         Person editedPerson = new PersonBuilder(personToEdit).withTags().build();
 
         assertEditSuccess(addressBookIndex, addressBookIndex, detailsToEdit, editedPerson);
     }
 
     @Test
     public void edit_findThenEdit_success() throws Exception {
-        commandBox.runCommand(FindCommand.COMMAND_WORD + " Elle");
+        commandBox.runCommand(FindCommand.COMMAND_WORD + " carl");
```
Comment (yamgent) [inline src/test/java/guitests/EditCommandTest.java]: Any reason why `carl` and `carrle` starts with lowercase?

**Comment by CanIHasReview** [comment]
# v3

@Zhiyuan-Amos submitted v3 for review.

* [v3 1/1] [Remove IndexUtil\#oneToZeroIndex\(int\)](https://github.com/se-edu/addressbook-level4/pull/463/commits/25ac925a40e30dac927731182a4d5882aa35a1d7)

([:books: Archive](https://github.com/se-edu/addressbook-level4/compare/dfe3b25eb274119f2d0605fa266877597a3341c6...25ac925a40e30dac927731182a4d5882aa35a1d7)) ([:chart_with_upwards_trend: Interdiff between v2 and v3](https://github.com/se-edu/addressbook-level4/compare/04b6350367a2f17337a8db3bbff492a7fb93c6aa~1...04b6350367a2f17337a8db3bbff492a7fb93c6aa))
<details>
<summary>Checkout this PR version locally</summary>

```sh
git fetch https://github.com/se-edu/addressbook-level4.git refs/pr/463/3/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you wish to fetch this PR to.

</details>

**Comment by Zhiyuan-Amos** [comment]
@yamgent for your review, and if possible to rope in external reviewers alrd :>

**Comment by Zhiyuan-Amos** [comment]
@chao1995 I've replied you a little on the other PR. :) 

The `ZeroBasedIndex` thing is implicit i.e `Model` takes in an `Index`, and internally it uses `int zeroBasedIndex` (i.e works similarly to `ZeroBasedIndex`).

**Pull request merged**